_includes/tutorials/anomaly-detection/ksql/markup/dev/find-suspicious-transactions.adoc

```diff
@@ -0,0 +1,11 @@
+Inspect the new stream.
+
++++++
+<pre class="snippet"><code class="sql">{% include_raw tutorials/anomaly-detection/ksql/code/tutorial-steps/dev/find-suspicious-transactions.sql %}</code></pre>
++++++
+
+Note that some of the transactions we inserted earlier were to companies that we flagged as possibly suspicious.
```

Comment:
"we flagged" -- did we flag them or is it just that their names were in the table?

Reply:
I altered this sentence to be more direct

> Note that some of the transactions we inserted earlier were to companies that are in the suspicious names table.